docs/index.rst

```diff
@@ -5,9 +5,14 @@ TChannel is a network multiplexing and framing protocol for RPC.
 
 The protocol is currently implemented in:
 
- * `Go`_
- * `Node`_
- * `Python`_
+ * Go
+    * `Guide <go-guide.html>`__
+    * `API Documentation <http://godoc.org/github.com/uber/tchannel/golang>`__
+ * Node
+    * `Guide <node-guide.html>`__
```

Comment:
does the Node.js community have a `godoc.org`-esque hosted documentation solution?